app/views/ingredients/index.html.erb

```diff
@@ -0,0 +1,5 @@
+<% @chef.dishes.each do |dish| %>
+  <% dish.ingredients.each do |ingredient| %>
+    <%= ingredient.name %> <br>
+  <% end%>
+<% end %>
```

Comment:
This is an N+1 query and should be avoided. I would move this work to the Chef model and create a method to grab all of the chef's ingredients from there and just call on that method in the view.